### PR TITLE
Add transform methods and vertex_centroid property

### LINKED ISSUE #7: Add units conversion method

### LINKED ISSUE #11: Add flip method

### LINKED ISSUE #12: Add centroid_of_vertices method

### PR DESCRIPTION
**Pull request by paulmelnikow**
Depends on lace/polliwog#173

Closes #7
Closes #11
Closes #12

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 numpy
 ounce>=1.1.0,<2.0
-polliwog==1.0.0b6
+-e git://github.com/lace/polliwog.git@b18ab1c07c9be3b9762ae105488a8800d0547c41#egg=polliwog
```
Comment (paulmelnikow) [inline requirements.txt]: This isn't intended to be merged; it should be replaced with a new release of polliwog.

Comment (jbeard4) [reply requirements.txt]: Understood

**Comment by paulmelnikow** [comment]
I released polliwog 1.0.0b7 and this is now pulling it in.

File: lacecore/_transform/transform_mixin.py [hunk 2]
```diff
@@ -11,21 +11,13 @@ def transform(self):
...
-            )
-            >>> upper_half_plus_right_half = (
-                mesh.select()
-                .vertices_above(centroid, dim=0)
-                .union()
-                .vertices_above(centroid, dim=1)
+            >>> transformed = (
+                mesh.transform()
+                .translate(3.0 * vg.basis.x)
+                .uniform_scale(3.0)
```
Comment (paulmelnikow) [inline lacecore/_transform/transform_mixin.py]: @jbeard4

**Pull request merged**